onnxruntime/python/onnxruntime_pybind_state_common.h

```diff
@@ -0,0 +1,44 @@
+#include "core/framework/allocator.h"
+#include "core/framework/session_options.h"
+#include "core/common/logging/logging.h"
+#include "core/common/logging/sinks/cerr_sink.h"
+
+namespace onnxruntime {
+namespace python {
+
+using namespace onnxruntime;
+using namespace onnxruntime::logging;
```

Comment:
avoid `using namespace` directives, per [https://google.github.io/styleguide/cppguide.html#Namespaces](https://google.github.io/styleguide/cppguide.html#Namespaces)

Reply:
actually it looks like we have an exception for that as long as it's not in the global scope: [https://github.com/microsoft/onnxruntime/blame/master/docs/Coding_Conventions_and_Standards.md#L19](https://github.com/microsoft/onnxruntime/blame/master/docs/Coding_Conventions_and_Standards.md#L19)

---
In reply to: [395253361](https://github.com/microsoft/onnxruntime/pull/3235#discussion_r395253361) [](ancestors = 395253361)